skrub/_dataframe/_common.py

```diff
@@ -71,12 +72,14 @@
     "is_null",
     "has_nulls",
     "drop_nulls",
+    "fill_nan",
```

Comment:
can we call it `fill_null` instead? to be consistend with `is_null`, `has_nulls`, `drop_nulls`, polars' `fill_null` and because we are replacing all nulls/missing values not only NaNs

Reply:
Renaming it `fill_null`